scripts/test_images.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+# http://redsymbol.net/articles/unofficial-bash-strict-mode/
+set -euo pipefail
+IFS=$'\n\t'
+
+source ./scripts/test_lib.sh
+source ./scripts/build_lib.sh
+
+function startContainer {
+    # run docker in the background
+    docker run -d --rm --name "${RUN_NAME}" "${IMAGE}"
+
+    # wait for etcd daemon to bootstrap
+    sleep 5
+}
+
+function runVersionCheck {
+    binary="/usr/local/bin/""$1"
+    versionString="$2"
+    Out=$(docker exec "${RUN_NAME}" "${binary}" "${versionString}")
+    foundVersion=$(echo "$Out" | head -1 | rev  | cut -d" "  -f 1 | rev )
+    if [[ "${foundVersion}" != "${VERSION}" ]]; then
+        echo "error: Invalid Version. Got $foundVersion, expected $VERSION. Error: $Out"
+        exit 1
+    fi
+}
+
+trap 'docker stop "${RUN_NAME}"' EXIT
```

Comment:
minor comment:  move it immediately after `startContainer`

Reply:
Done.

Reply:
I mean moving it immediately after line 66.